tests/framework/utils.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Maybe have been nicer to just have a function that would run a command, then verify with a regexp and use retry_call - https://pypi.org/project/retry/ for handling the retry part?

Reply:
I like @raduiliescu s suggestion. We already use `retry_call` in a bunch of places: https://github.com/firecracker-microvm/firecracker/blob/master/tests/framework/jailer.py#L201, https://github.com/firecracker-microvm/firecracker/blob/master/tests/framework/microvm.py#L357. We could stick to it across the CI framework.

Reply:
I updated the PR to use the reply module.